fluid-project/kettle
# KETTLE-71 - Add Vagrantfile

First pass to adding Kettle to Jenkins CI using the standard Vagrant setup we have for other projects.

How to test:

* Checkout PR
* `vagrant up` # `npm audit` warnings will show in red, it's expected.
* `npm run lint:vagrant`
* `npm run test:vagrant`
